packages/blunderbuss/test/blunderbuss.ts

```diff
@@ -710,14 +747,19 @@ describe('Blunderbuss getConfigWithDefault', () => {
 
     scope.done();
 
-    validateConfigStub.calledOnceWith(
-      sinon.match.instanceOf(Octokit),
+    sinon.assert.calledOnceWithExactly(
+      validateConfigStub,
+      sinon.match.instanceOf(ProbotOctokit),
       'testOwner',
       'testRepo',
       'c5b0c82f5d58dd4a87e4e3e5f73cd752e552931a',
       6
     );
-    assignStub.calledOnceWith(sinon.match.instanceOf(Context), sinon.match.any);
+    sinon.assert.calledOnceWithExactly(
+      assignStub,
+      sinon.match.instanceOf(Context),
+      sinon.match.any
```

Comment:
Seems strange that we're confirming whether or not something was called with 'any', I feel like it might defeat the purpose of the method. But not a blocker!

Reply:
I changed it to sinon.match.object, a slightly narrower version. PTAL